docs/testplan/Smartswitch-test-plan.md

```diff
@@ -0,0 +1,876 @@
+        PSU1 -- not present
+
+Cooling Devices
+    fantray0            FAN-2RU-PI-V3   N/A                  N/A             Cisco 8000 Series 2RU Fan with Port-side Air Intake Ver 3
+    fantray1            FAN-2RU-PI-V3   N/A                  N/A             Cisco 8000 Series 2RU Fan with Port-side Air Intake Ver 3
+    fantray2            FAN-2RU-PI-V3   N/A                  N/A             Cisco 8000 Series 2RU Fan with Port-side Air Intake Ver 3
+    fantray3            FAN-2RU-PI-V3   N/A                  N/A             Cisco 8000 Series 2RU Fan with Port-side Air Intake Ver 3
+
+FPDs
+    RP0/info.0                          0.8.0-287                            \_SB_.PC00.RP07.PXSX.INFO
+    RP0/info.1                          0.4.7-122                            \_SB_.PC00.RP01.PXSX.INFO
+    RP0/info.2                          0.2.1-247                            \_SB_.PC00.RP10.PXSX.INFO
+    RP0/info.50.auto                    10.2.0-30                            \_SB_.PC00.RP07.PXSX.P2PF
+
+root@sonic:/home/cisco# config chassis modules startup DPU4
+
+```
+#### Pass/Fail Criteria
+ * Verify dpu powered off in platform inventory after dpu shut down
+ * Verify dpu is shown in platform inventory after dpu powered on
+
+
+### 1.7 Check removal of pcie link between npu and dpu
+
+#### Steps
+ * Use command `pcieutil generate` to generate pcie yaml
+ * Use `show platform pcieinfo -c` to run the pcie info test to check everything is passing
+ * Use command `echo 1 > /sys/bus/pci/devices/BUS_ID/remove` to remove pcie link between npu and one dpu
+ * Use `show platform pcieinfo -c` to run the pcie info test to check pcie link has been removed
+ * Use command `echo 1 > /sys/bus/pci/rescan` to rescan pcie links
+ * Use `show platform pcieinfo -c` to run the pcie info test to check everything is passing
+ * This test is to check the PCie hot plug functinality since there is no OIR possible
+
+#### Verify in
+ * Switch
+   
+#### Sample Output
+```
+On Switch: Showing example of one dpu pcie link
+
+root@sonic:/home/cisco# pcieutil generate
+Are you sure to overwrite config file pcie.yaml with current pcie device info? [y/N]: y
+Generated config file '/usr/share/sonic/device/x86_64-8102_28fh_dpu_o-r0/pcie.yaml'
+root@sonic:/home/cisco# show platform pcieinfo -c
+
+root@sonic:/home/cisco# echo 1 > /sys/bus/pci/devices/0000:1a:00.0/remove
+root@sonic:/home/cisco# 
+root@sonic:/home/cisco# echo 1 > /sys/bus/pci/rescan
+root@sonic:/home/cisco# 
+root@sonic:/home/cisco# show platform pcieinfo -c
+
+```
+#### Pass/Fail Criteria
+ * Verify after removing pcie link, pcie info test fail only for that pcie link.
+ * Verify pcieinfo test pass for all after bringing back up the link
+
+
+### 1.8 Check the NTP date and timezone between DPU and NPU
+
+#### Steps
+ * In Switch, under the file /etc/ntp.conf configure it to use the ntp server and restart ntp.service to configure
```

Comment:
You are right. This test case is just to check that both NPU and DPU are in sync with the dates. Nothing to do with any configurations

Reply:
Can you please update the steps? Because the first step here is describes that the configuration will be set:

> under the file /etc/ntp.conf configure it to use the ntp server and restart ntp.service

Reply:
Changed it.